public/app/features/explore/ExploreTimeControls.tsx

```diff
@@ -67,18 +70,35 @@ export class ExploreTimeControls extends Component<Props> {
   };
 
   render() {
-    const { range, timeZone } = this.props;
+    const { range, timeZone, splitted, syncedTimes, onClick } = this.props;
+    const syncButton = { icon: 'fa fa-link', onClick, active: syncedTimes };
 
     return (
-      <TimePicker
-        value={range}
-        onChange={this.onChangeTimePicker}
-        timeZone={timeZone}
-        onMoveBackward={this.onMoveBack}
-        onMoveForward={this.onMoveForward}
-        onZoom={this.onZoom}
-        selectOptions={this.setActiveTimeOption(defaultSelectOptions, range.raw)}
-      />
+      <>
+        {splitted && (
+          <TimePicker
```

Comment:
Since you are rendering the same component depending on the condition I suggest refactoring this to sth like this:

```tsx
const timePickerProps = 
  splitted ? {
    // splitted props
    ...
  } : {
    // non-splitted props
    ...
  };

return <TimePicker {...timePickerProps} />
```

Reply:
This implementation looks so much cleaner 😍!